portal-web/test/functional/com/liferay/portalweb/macros/CompanyLogging.macro

```diff
@@ -0,0 +1,89 @@
+definition {
+
+	macro assertCompanyLogIsolated {
+		if ("${portalInstanceName}" == "liferay.com") {
```

Comment:
It would be better to add a macro here into JSONCompany.macro since i saw this reused a bit